tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -0,0 +1,26 @@
+class God {
+  static create() {
+    return [new Man('Adam', 'male'), 
+      new Woman('Eve', 'female')];
+  }
+}
+
+class Human {
+  constructor(name, sex) {
+    this.name = name;
+    this.sex = sex;
+  }
+}
+
+class Man extends Human {
+  constructor(name, sex) {
```

Comment:
Isn't `sex` is defined for `Man` and `Woman`? Why do you need to pass this as arguments?